deeplabcut/pose_tracking_pytorch/model/backbones/vit_pytorch.py

```diff
@@ -346,12 +346,12 @@ def resize_pos_embed(posemb, posemb_new, hight, width):
     gs_old = int(math.sqrt(len(posemb_grid)))
     print(
         "Resized position embedding from size:{} to size: {} with height:{} width: {}".format(
-            posemb.shape, posemb_new.shape, hight, width
+            posemb.shape, posemb_new.shape, height, width
         )
     )
     posemb_grid = posemb_grid.reshape(1, gs_old, gs_old, -1).permute(0, 3, 1, 2)
-    posemb_grid = F.interpolate(posemb_grid, size=(hight, width), mode="bilinear")
-    posemb_grid = posemb_grid.permute(0, 2, 3, 1).reshape(1, hight * width, -1)
+    posemb_grid = F.interpolate(posemb_grid, size=(height, width), mode="bilinear")
+    posemb_grid = posemb_grid.permute(0, 2, 3, 1).reshape(1, height * width, -1)
```

Comment:
Changes in `deeplabcut/pose_tracking_pytorch/model/backbones/vit_pytorch.py` are source changes